packages/openapi-generator/src/cli.ts

```diff
@@ -25,10 +25,11 @@ export = class GenerateOpenApiClient extends Command {
   ).version;
 
   static flags = {
-    inputDir: flags.string({
-      name: 'inputDir',
+    input: flags.string({
```

Comment:
I think that name makes more sense than the old one as it also allows to handle files (without directory). However, this now differs from the OData generator. We should consider renaming (and deprecating the old one) it there too. What do you think @FrankEssenberger @jjtang1985?